documentation/docs/02-layouts.md

```diff
@@ -62,6 +62,10 @@ We can create a layout that only applies to pages below `/settings` (while inher
 <slot></slot>
 ```
 
+### Resets
+
+To reset the layout stack, create a `$layout.reset.svelte` file. For example, if you want your `/admin/*` pages to _not_ inherit the root layout, create a file called `src/routes/admin/$layout.reset.svelte`.
```

Comment:
We should say something about the contents of `$layout.reset.svelte`. Would it be accurate to say:

> To reset the layout stack, create an empty `$layout.reset.svelte` file.

Reply:
To me this sounds like it needs to be empty, which is not correct because you can starter writing your new layout inside the reset file right away. Do I think the current sentence is better.

Reply:
I'm not sure what would be best but I agree that the current proposed docs are a bit confusing. Maybe it would help if they said one creates the reset file _instead_ of the regular layout file?

Reply:
Made it a bit clearer